app/mailers/notify.rb

```diff
@@ -1,6 +1,5 @@
 class Notify < ActionMailer::Base
   include ActionDispatch::Routing::PolymorphicRoutes
-
```

Comment:
Can you revert this line? Since it does not add any new code its a shame to have it in the git history :)